test/production/app-dir/actions-tree-shaking/use-effect-actions/use-effect-actions-edge.test.ts

```diff
@@ -0,0 +1,3 @@
+process.env.TEST_EDGE = '1'
+
+require('./use-effect-action.test')
```

Comment:
idk about this... feel like it'd be nicer to just do something like
```
describe.each(['nodejs', 'edge'])('actions-tree-shaking - %s - use-effect-actions', (runtime) => {
  ...
})
```

Reply:
it's not convenient for running single test for certain runtime and work with jest filter arg `-t`.

Reply:
this always worked for me: `pnpm test-dev ... -t '.*- edge -.*my test'`